lib/trino-hive-formats/src/test/java/io/trino/hive/formats/ion/TestIonFormat.java

```diff
@@ -260,6 +300,216 @@ private void assertValues(RowType rowType, String ionText, List<Object>... expec
         }
     }
 
+    private static Deserializer createHiveDeserializer(List<Column> columns, boolean failOnOverflow)
```

Comment:
Based on the offline discussion, we are not planning to support the` ion.fail_on_overflow` config in the Trino connector. In this case, we might need to declare the value of `failOnOverflow` as needed instead of passing it.